packages/bpk-component-image/src/BpkImage.scss

```
@@ -83,4 +83,8 @@
   &--no-background {
     @include bpk-image--no-background;
   }
+
+  &--borderRadius-sm {
```

We use kebab case in the rest of this file so we should preserve it.

```suggestion
  &--border-radius-sm {
```

---

Done :)